core/org.osate.aadl2.modelsupport/src/org/osate/aadl2/modelsupport/resources/OsateResourceUtil.java

```diff
@@ -111,10 +113,19 @@ public static XtextResourceSet getResourceSet() {
 
 	}
 
+	/**
+	 * @deprecated
```

Comment:
Will this be removed?
```suggestion
	 * @deprecated will be removed in 2.5.0
```